src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/ArticleHeader.tsx

```diff
@@ -70,13 +98,23 @@ const Img = styled.div<{ src?: string }>`
     mix-blend-mode: screen;
     filter: grayscale(100%);
   `};
+
+  ${props =>
+    props.isVisible &&
+    `
+      opacity: 1;
+    `};
 `
 
-const GridItem = styled(Box)`
+const GridItem = styled(Box)<{ isVisible?: boolean }>`
   border: 3px solid ${color("purple100")};
-  width: 25%;
   transition: background-color 0.5s;
 
+  ${props =>
+    props.isVisible &&
+    `
```

Comment:
Not sure why, but attempted to change the Img opacity in this block and it would not change... added an isVisible prop above as well.

Reply:
Maybe that's related to the definition order? Have you tried moving this chunk to the end of the ``styled`...` `` block?